.github/workflows/playwright.yml

```
@@ -0,0 +1,61 @@
+      uses: actions/cache/save@v4
+      with:
+        path: ${{ matrix.directory }}/node_modules
+        key: ${{ runner.os }}-${{ matrix.directory }}-${{ hashFiles(format('{0}/yarn.lock', matrix.directory)) }}
+
+  test:
+    needs: install-dependencies
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Restore cached node_modules for Playwright
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 4

(indentation)